venues/OpenReview.net/Support/init.py

```diff
@@ -167,3 +167,46 @@
         'content': request_content
     }
 }))
+
+comment_inv = client.post_invitation(openreview.Invitation(**{
+    'id': 'OpenReview.net/Support/-/Comment',
+    'readers': ['everyone'],
+    'writers': ['OpenReview.net/Support'],
+    'signatures': ['OpenReview.net/Support'],
+    'invitees': ['everyone'],
+    'process': 'commentProcess.js',
+    'reply': {
+        'forum': None,
+        'replyto': None,
+        'invitation': 'OpenReview.net/Support/-/Request_Form',
+        'readers': {
+            'description': 'Select all user groups that should be able to read this comment.',
+            'values': ['OpenReview.net/Support']
```

Comment:
With this only value there is not way to reply to the contact emails.

Reply:
Good point. I tried to make a quick fix for this, but I don't think we can get the contact email values from the request form in the comment readers field unless we create a group for them.

(in the request form itself, we can just do values-copied from the Contact Emails field)

Reply:
Does it work having one invitation for all the notes? May be we need to create a comment invitation per forum and add the contact emails there.